query/src/test/scala/hmda/query/model/institutions/InstitutionQueryGenerators.scala

```diff
@@ -7,20 +7,60 @@ object InstitutionQueryGenerators {
   implicit def institutionQueryGen: Gen[InstitutionQuery] = {
```

Comment:
One possible usage might be converting from `InstitutionQuery` to `Institution`, though I don't think we are doing that right now (or have a need to in the immediate future)